packages/core/src/transactions.ts

```diff
@@ -88,12 +89,26 @@ export function getEncodeObjectFromMultiChainMessageInjective(
   throw new Error("Unsupported message type");
 }
 
-export function getGasAmountForMessage(message: MultiChainMsg) {
-  if (message.msgTypeURL === "/cosmwasm.wasm.v1.MsgExecuteContract") {
-    if (message.chainID === "neutron-1") {
+export async function getGasAmountForMessage(
+  client: SigningCosmWasmClient,
+  signerAddress: string,
+  message: MultiChainMsg,
+) {
+  if (
+    message.chainID.includes("evmos") ||
+    message.chainID.includes("injective")
+  ) {
+    if (message.msgTypeURL === "/cosmwasm.wasm.v1.MsgExecuteContract") {
       return "2400000";
     }
-    return "2000000";
+    return "280000";
   }
```

Comment:
I'd like to get rid of these hardcoded values for evmos and injective but it will need to happen when I have more time to give it proper attention